micro_sam/training/training.py

```diff
@@ -146,6 +147,8 @@ def train_sam(
     scheduler_kwargs: Optional[Dict[str, Any]] = None,
     save_every_kth_epoch: Optional[int] = None,
     pbar_signals: Optional[QObject] = None,
+    optimizer_class: Optional[Optimizer] = torch.optim.Adam,
```

Comment:
I would switch to AdamW as the default here too.

Reply:
Okie, I will switch it right away.

Just to keep you posted, the current `micro-sam` models (including the generalists) have been trained using `Adam`.